panther_analysis_tool/constants.py

```diff
@@ -14,7 +14,9 @@
     SCHEDULED_QUERY_SCHEMA,
 )
 
-VERSION_STRING: Final = "panther_analysis_tool 0.20.0"
+PACKAGE_NAME: Final = "panther_analysis_tool"
+
+VERSION_STRING: Final = "0.20.0"
```

Comment:
Bump version?

Reply:
I noticed we didn't release with the last merge (happy to walk through that, its not automated... yet).  So I think its fine to keep this version and cut it after.